cmd/crc/cmd/start.go

```diff
@@ -144,12 +144,12 @@ func checkIfNewVersionAvailable(noUpdateCheck bool) {
 	}
 	isNewVersionAvailable, newVersion, err := version.NewVersionAvailable()
 	if err != nil {
-		logging.Debugf("Error checking if a new version is available: %v", err)
+		logging.Debugf("Unable to find out if a new version is available: %v", err)
 		return
 	}
 	if isNewVersionAvailable {
 		logging.Warnf("A new version (%s) has been published on %s", newVersion, constants.CrcLandingPageURL)
 		return
 	}
-	logging.Debugf("No new version available, latest version at mirror is %s", newVersion)
+	logging.Debugf("No new version available, the latest version is %s", newVersion)
```

Comment:
`No new version available. The latest version is ...`

needs to be a full-stop